README.md

```diff
@@ -97,6 +97,10 @@ For documentation on how to configure code-gov-front-end, go [here](CONFIGURATIO
 ## Questions?
 
 If you have questions, please feel free to [send us an email](mailto:code@gsa.gov).
+Twitter: @CodeDotGov
```

Comment:
Adding 2 spaces to the end of the line should make it force a line break. Try this on each of these lines you added.